plugins/node/instrumentation-undici/README.md

```diff
@@ -41,20 +43,19 @@ provider.register();
 registerInstrumentations({
   instrumentations: [new UndiciInstrumentation()],
 });
-
 ```
 
 ### Undici/Fetch instrumentation Options
 
 Undici instrumentation has few options available to choose from. You can set the following:
 
-| Options | Type | Description |
-| ------- | ---- | ----------- |
-| [`ignoreRequestHook`](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/node/opentelemetry-instrumentation-undici/src/types.ts#63) | `IgnoreRequestFunction` | Undici instrumentation will not trace all incoming requests that matched with custom function. |
-| [`requestHook`](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/node/opentelemetry-instrumentation-undici/src/types.ts#65) | `RequestHookFunction` | Function for adding custom attributes before request is handled. |
-| [`startSpanHook`](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/node/opentelemetry-instrumentation-undici/src/types.ts#67) | `StartSpanHookFunction` | Function for adding custom attributes before a span is started. |
-| [`requireParentforSpans`](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/node/opentelemetry-instrumentation-undici/src/types.ts#69) | `Boolean` | Require a parent span is present to create new span for outgoing requests. |
-| [`headersToSpanAttributes`](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/node/opentelemetry-instrumentation-undici/src/types.ts#71) | `Object` |  List of case insensitive HTTP headers to convert to span attributes. Headers will be converted to span attributes in the form of `http.{request\|response}.header.header-name` where the name is only lowercased, e.g. `http.response.header.content-length`|
+| Options                                                                                                                                                | Type                    | Description                                                                                                                                                                                                                                                  |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
You have a formatter that does this for you, do you?

nit: I'm not sure this helps readablility for those reading README.md in a text editor. ;)
However, I know why it is being done.

Reply:
like the code changes above, this was also formatted via a markdown extension for vscode that I have enabled by default for all JS related projects.

here's what this looks like in the plain diff

<img width="738" alt="image" src="https://github.com/open-telemetry/opentelemetry-js-contrib/assets/18689448/a2928107-9427-4430-98ba-a7271403a10c">

yeah unfortunately markdown formatting here is super subjective, I know other formatters take a different approach.